workflows/articNcovNanopore.nf

```diff
@@ -61,8 +61,8 @@ workflow sequenceAnalysisNanopolish {
                            .join(articRemoveUnmappedReads.out))
 
      if (params.outCram) {
-        bamToCram(qc.pass.map{ it[0] } 
-                        .join (trimPrimerSequences.out.ptrim.combine(ch_preparedRef.map{ it[0] })) )
+        bamToCram(articMinIONNanopolish.out.ptirm.map{ it[0] } 
```

Comment:
Is that a typo for ptrim? Should it be trimPrimerSequences.out.ptrim?

Reply:
This is Nanopore workflow,  do I add bamToCram there or just keep it to Illumina?

Reply:
I guess you haven't tested the nanopore workflow, but I think if you can have it work in theory it's worth doing it. I think we want to use nanopore workflow as well, but I don't have any test data.

Reply:
Should work for nanopore bam as well just need to decide which bam files need to convert

Reply:
I'd say, convert the same bams that are used as input to QC.

Reply:
yes, that's what I thought as well and used same input files as QC:
makeQCCSV(articMinIONNanopolish.out.ptrim,.....)

Reply:
Ok, well it is still a typo. You've put ptirm, not ptrim.

Reply:
thanks, corrected now